modules/developer_manual/examples/core/webdav_api/files_versions/successful-response-which-contains-files.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<d:multistatus xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns" xmlns:oc="http://owncloud.org/ns">
+  <d:response>
+    <d:href>/remote.php/dav/meta/43/v/</d:href>
+    <d:propstat>
+      <d:prop>
+        <d:resourcetype>
+          <d:collection/>
+        </d:resourcetype>
+      </d:prop>
+      <d:status>HTTP/1.1 200 OK</d:status>
+    </d:propstat>
+  </d:response>
+  <d:response>
+    <d:href>/remote.php/dav/meta/43/v/1576663297</d:href>
```

Comment:
might be nice to add at least a second file, to illustrate that there can be more than one ?

Reply:
Did you mean "at least a second version"? I'm guessing that that's what you meant.

Reply:
I just updated the content with an example that shows a file with multiple versions.